.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
     - TRAVIS_FLAVOR=mysql FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=twemproxy FLAVOR_VERSION=latest
     - TRAVIS_FLAVOR=tokumx
+    - TRAVIS_FLAVOR=directory FLAVOR_VERSION=latest
```

Comment:
Small nit, since this test doesn't have a version, can you remove `FLAVOR_VERSION`? We use that to describe the version of the package installed, e.g. Mongo 3.2.